Apps/WebAPI/sdLitica.WebAPI/Controllers/v1/TimeseriesController.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using sdLitica.WebAPI.Services;
+
+namespace sdLitica.WebAPI.Controllers.v1
+{
+    /// <summary>
+    /// This controller is used to work with user timeseries
+    /// </summary>
+    [Route("api/v1/timeseries")]
+    //TODO: Change auth settings here
+    [AllowAnonymous]
+    public class TimeseriesController : BaseApiController
+    {
+        private readonly IInfluxDB _influxDb;
```

Comment:
Please, you should reference a ``ITimeSeriesService`` from ``sdLitica.Service`` component, inside a Controller.